README.rst

```diff
@@ -186,3 +186,5 @@ Common problems and fixes
 =========================
 * libXp.so.6 missing (in ubuntu >= 15.10, for example)
 This is a known is http://askubuntu.com/questions/719839/libxp-so-6-missing-15-10. The idea is to install it manually from official sources https://launchpad.net/ubuntu/wily/+package/libxp6
+
+* Beware: the pathname for "scratch" (directory for cache) must not contain any space character! This triggered a nasty bug in nipype....
```

Comment:
On Sun, Jun 19, 2016 at 2:50 PM, Christophe Pallier <
notifications@github.com> wrote:

> In README.rst
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67616456>:
>
> > @@ -186,3 +186,5 @@ Common problems and fixes
> >  =========================
> >  * libXp.so.6 missing (in ubuntu >= 15.10, for example)
> >  This is a known is http://askubuntu.com/questions/719839/libxp-so-6-missing-15-10. The idea is to install it manually from official sources https://launchpad.net/ubuntu/wily/+package/libxp6
> > +
> > +* Beware: the pathname for "scratch" (directory for cache) must not contain any space character! This triggered a nasty bug in nipype....
>
> I need to learn how to do better PR ;-) I only have one branch on my fork
> of pypreprocess. Should I have created different branches for different
> dev? Chris
>
Yes, you want to create different branches for handling different bugs or
feature requests. This way, you can hop from one branch to another, without
running out of sync. Also, ideally, you never work on the master branch :)
. OK roughly, the basic workflow for doing PRs is:
- you create a new local branch for each new feature you're implementing.
This may be implementing a new feature, or fixing a small group (possibly a
singleton) of tightly linked bugs, etc.
   So, you'd run "pull --rebase upstream master && checkout master &&
checkout -b branch-for-feature-xyz". In your local git configuration
(.git/config), "upstream" should point to the official
neurospin/pypreprocess repo, whilst "origin" should point to your forked
version on github (e.g, mine is https://github.com/dohmatob/pypreprocess).
- git push -u origin branch-for-feature-xyz, to forward your local changes
to github
- on your github pypreprocess repo, github will propose to send a new PR
neurosin/master <== chris/branch-for-cool-stuff. Confirm, and that's it.
- do this for any new feature you're implementing, or bug (or small group
of) you're trying to fix.
Hope this helps.

> … <#m_1057442999557563642_>
> -- Christophe Pallier Contact info:
> http://www.pallier.org/w/pmwiki.php/Main/Contact
> On Sun, Jun 19, 2016 at 1:15 PM, DOHMATOB Elvis ***@***.***> wrote: In
> README.rst <#231 (comment)
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67615081>>:
> > @@ -186,3 +186,5 @@ Common problems and fixes > =========================
> > * libXp.so.6 missing (in ubuntu >= 15.10, for example) > This is a known
> is http://askubuntu.com/questions/719839/libxp-so-6-missing-15-10. The
> idea is to install it manually from official sources
> https://launchpad.net/ubuntu/wily/+package/libxp6 > + > +* Beware: the
> pathname for "scratch" (directory for cache) must not contain any space
> character! This triggered a nasty bug in nipype.... Ideally, this would go
> in a separate PR as it has nothing to do with spm12 standalon installation
> or STC. — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub <
> https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67615081>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABdbMoFkN-AlBR_VpaTi2BpGIwM8hV9Tks5qNSTjgaJpZM4IqgU1>
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67616456>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAms1NB24AA9F3Xnoh_mviVwwl-sIsIcks5qNTsxgaJpZM4IqgU1>
> .
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob